lib/ipmi_backend_utils.pm

```diff
@@ -303,4 +303,26 @@ sub set_serial_console_on_vh {
 
 }
 
+#Huawei arm machines require password login after "boot from local disk" is selected.
+#Before the desired grub menu is shown, you need to navigate into the boot menu and
+#select the "sles" boot item.
+sub boot_local_disk_arm_huawei {
+    my $type_speed = 20;
+
+    assert_screen('input-password-huawei', 180);
+    type_string "Huawei12#\$", $type_speed;
```

Comment:
We have wrappers to type slow and very slow...  :) those might suit very well your use case. 

```suggestion
    type_string_slow "Huawei12#\$";
```